### PR TITLE
allow popover to pass arrow prop down to tippyjs

### PR DESCRIPTION
**Pull request by Dantiff**
Allow popover to pass arrow prop down to tippyjs

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @ben-dalton, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

**Comment by ben-dalton** [comment]
@Dantiff I believe line 37, 41, 45, and 49 should refer to the popover background color rather than white.

File: src/Popover/Popover.component.tsx [hunk 1]
```diff
@@ -56,10 +56,10 @@ export class Popover extends React.Component<Props> {
     theme: Themes.defaultTheme,
   };
   public render() {
-    const { theme, arrow, children, ...props } = this.props;
+    const { theme, children, ...props } = this.props;
     const withArrowDistance = 10;
     const withoutArrowDistance = 9;
-    const distance = arrow ? withArrowDistance : withoutArrowDistance;
+    const distance = props.arrow ? withArrowDistance : withoutArrowDistance;
```
Comment (ben-dalton) [inline src/Popover/Popover.component.tsx]: It looks like there is not enough distance when the arrow is present.

Comment (ben-dalton) [reply src/Popover/Popover.component.tsx]: It's not clear in the design document, but I believe the tip of the arrow should be the same distance from the button as the top of the box in this design without the arrow. 

Screen Shot 2019-04-11 at 11 55 05 AM

Perhaps the distance should be an optional prop that could override this distance as well?

Comment (ben-dalton) [reply src/Popover/Popover.component.tsx]: actually, I see that distance is already an optional prop. 👍

**Comment by ben-dalton** [comment]
@Dantiff Can we move the popover and tooltip theme properties that are not related to typography out of the typography section - up one level?

**Comment by Dantiff** [comment]
@ben-dalton I have made updates as agreed upon

**Pull request merged**